### PR TITLE
[zh-cn] Fix links in access-authn-authz page

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

File: content/zh-cn/docs/reference/access-authn-authz/_index.md [hunk 1]
```diff
@@ -16,8 +16,8 @@ read [Controlling Access to the Kubernetes API](/docs/concepts/security/controll
 
 Reference documentation:
 -->
-关于 Kubernetes 如何实现和控制 API 访问的介绍性材料，可阅读
-[控制 Kubernetes API 的访问](/zh/docs/concepts/security/controlling-access/)。
+关于 Kubernetes 如何实现和控制 API 访问的介绍性材料，
+可阅读[控制 Kubernetes API 的访问](/zh/docs/concepts/security/controlling-access/)。
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/access-authn-authz/_index.md:20]:
```suggestion
可阅读[控制 Kubernetes API 的访问](/zh-cn/docs/concepts/security/controlling-access/)。
```

Comment (mengjiao-liu) [inline content/zh-cn/docs/reference/access-authn-authz/_index.md:20]: Why not change this link to `zh-ch` prefix? Anything special?

**Pull request merged**